test/collection.json

```diff
@@ -512,7 +512,7 @@
 					"formdata": [
 						{
 							"key": "operations",
-							"value": "{\"query\": \"mutation InputWithUploadArray($input: InputWithUploadArray!, $input2: [NestedInput!]!) {inputWithUploadArray(input: $input, input2: $input2) {filename,mimetype,encoding,content}}\", \"variables\": {\"input2\": [{\"str\": \"str\", \"nested\": {\"str\": \"nested str\"}}], \"input\": {\"str\": \"a string\"}}}",
+							"value": "{\"query\": \"mutation InputWithUploadArray($input: InputWithUploadArray!, $input2: [NestedInput!]!) {inputWithUploadArray(input: $input, input2: $input2) {filename,mimetype,encoding,content}}\", \"variables\": {\"input2\": [{\"str\": \"str\", \"nested\": {\"str\": \"nested str\", \"f\": null}}], \"input\": {\"str\": \"a string\", \"files\": []}}}",
```

Comment:
The input fields that the files map to need to actually exist in the mutation for `graphql-upload-minimal`. That's the only real change in switching out deps.